resources/js/components/journal/partials/JournalContentEntry.vue

```diff
@@ -30,6 +31,11 @@
             <h3 class="mb1">
               {{ entry.title }}
             </h3>
+            <p class="mb1">
+              <span class="pr2 f6 avenir">
+                {{ $t('journal.journal_created_at', { date: entry.created_at }) }}
+              </span>
+            </p>
```

Comment:
The date is now visible for each journal log, and this can be deleted
```suggestion
```

Reply:
Done!